README.rst

```diff
@@ -8,15 +8,17 @@ Let's Encrypt App for Django
    :target: https://codecov.io/gh/urda/django-letsencrypt/branch/master
 
 :code:`django-letsencrypt` will allow you to add, remove, and update any
-`ACME challenges <https://letsencrypt.github.io/acme-spec/>`_ objects you may
+`ACME challenge <https://letsencrypt.github.io/acme-spec/>`_ objects you may
 need through your Django admin interface. Simply add the :code:`ACME challenge`
-and :code:`response`, and your app will serve up the necessary URL for
+and :code:`response` for your app to serve up the necessary information for
 `Let\'s Encrypt <https://letsencrypt.org/how-it-works/>`_ validation.
 
-Quick Start
------------
+Installation & Configuration
+----------------------------
 
-1. Add :code:`letsencrypt` to your :code:`INSTALLED_APPS`
+1. `pip install django-letsencrypt`
```

Comment:
This isn't right at all. `rst` do not use `md` style in-line code blocks. 

Needs to be:

```rst
:code:`pip install django-letsencrypt`
```